src/components/AuthForm/AuthForm.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useRef, useState } from "react";
+import { useStore, useDispatch } from "react-redux";
+
+import { fetchLoginData, fetchSignUpData } from "../../ducks/userInfo";
+
+import GoogleSrc from '../../assets/google-symbol.svg';
+
+import {
+    AuthFormContainer,
+    AuthMainContainer,
+    AuthEnterWithGoogle,
+    GoogleIcon,
+    AuthText,
+    InputLabel,
+    ErrorStar,
+    Input,
+    ErrorMsg,
+    AuthActionButtons,
+    Button
+} from "./AuthForm.style";
+
+let formData = {email: null, password: null};
```

Comment:
![image](https://user-images.githubusercontent.com/52242019/184375188-0767f6e3-c880-4525-9ed7-942098b19600.png)
const is quite better